src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -171,76 +196,80 @@ def _find_resources(self, node, requested_cores, requested_gpus,
         NOTE: chunking is only applied to cores at this point.
 
         When `partial` is set to `True`, this method is allowed to return
-        a *partial* match, so to find less cores and gpus then requested (but
-        the call will never return more than requested).
+        a *partial* match, so to find less cores, gpus, and local_fs then
+        requested (but the call will never return more than requested).
         '''
 
         # list of core and gpu ids available in this node.
         cores = list()
-        gpus  = list()
+        gpus = list()
+        lfs = None
 
-        # first count the number of free cores and gpus. This is
-        # way quicker than actually finding the core IDs.
+        # first count the number of free cores, gpus, and local file storage.
+        # This is way quicker than actually finding the core IDs.
         free_cores = node['cores'].count(rpc.FREE)
```

Comment:
You can't really `count the number of ... file storage` ...

Reply:
Not sure if this still applies?